app/src/main/java/com/example/pinterbest/App.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.pinterbest
+
+import android.app.Application
+import com.example.pinterbest.di.ApplicationComponent
+import com.example.pinterbest.di.DaggerApplicationComponent
+import com.example.pinterbest.presentation.common.ApplicationAPI
+import com.example.pinterbest.presentation.common.ApplicationComponentHolder
+
+class App : Application(), ApplicationComponentHolder {
+
+    private lateinit var appComponent: ApplicationComponent
+
+    override fun getApplicationAPI(): ApplicationAPI {
```

Comment:
I suggest to keep `camelCase` even for abbreviations (`ApplicationApi`)

Reply:
Fixed